.github/workflows/e2e-manual.yml

```diff
@@ -10,13 +11,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        group: ['00', '01', '02', '03']
+        group: ['00', '01', '02', '03', '04']
```

Comment:
I'm not sure this makes sense to backport: the overhead of each additional group is about 5 minutes, and if the number of tests in each group is low enough you end up just spending a lot of time, for sort of minimal overhead. There are many fewer 0.34.x-era e2e tests (because there are fewer dimensions in the matrix.)

We could expand coverage, but I don't know that blind porting this gets us very much without a little more thought.

Reply:
We're planning on forward-porting our current E2E test suite to v0.37. Does that change your view on this at all?

Reply:
Potentially. It's really about the number of test cases generated and the expected runtime of the suite: and it's easy enough to change it later when we've actually done that port, and there's a lot of overhead per-group. I don't think this is should block.

Reply:
I've reverted this.